docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+
+services:
+  redis:
+    image: redis:5.0-alpine
+    ports:
+      - "6379:6379"
+    entrypoint: redis-server --appendonly yes
```

Comment:
If this file is not mentioned in the README in any way, I think we must remove it.
We cannot keep files in the repositories which do not serve a specific purpose and are not explained anywhere.